client/postcss.config.js

```diff
@@ -1,8 +1,19 @@
-module.exports = () => ( {
-	plugins: {
-		'postcss-custom-properties': {
-			importFrom: [ require.resolve( '@automattic/calypso-color-schemes' ) ],
+const config = require( './server/config' );
+const bundleEnv = config( 'env' );
+
+if ( bundleEnv === 'development' ) {
```

Comment:
All of our evergreen browsers definitely support CSS custom properties, so you can certainly make use of that here. You'll need something like what we do in `webpack.config.js`:

```js
const isCalypsoClient = process.env.BROWSERSLIST_ENV !== 'server';
const isDesktop = calypsoEnv === 'desktop' || calypsoEnv === 'desktop-development';
const defaultBrowserslistEnv = isCalypsoClient && ! isDesktop ? 'evergreen' : 'defaults';
const browserslistEnv = process.env.BROWSERSLIST_ENV || defaultBrowserslistEnv;

if ( browserslistEnv === 'defaults' ) {
```

That said, I don't like the duplication of logic. What do you thing is the best way of handling things here, @jsnajdr? Should `webpack.config.js` set the `BROWSERSLIST_ENV` environment variable to the value of `browserslistEnv` so that subprocesses like `postcss` can reuse it?

Reply:
The webpack config can pass some arbitrary options down to the `postcss-loader` and in turn to the `postcss.config.js` file. They call it [context](https://github.com/postcss/postcss-loader#context-ctx).

The [code that configures the `SassLoader`](https://github.com/Automattic/wp-calypso/blob/1b61ce622ae3c975585d3dbbe461eeaed9cf5a78/client/webpack.config.js#L196-L200) will look like this:
```js
SassConfig.loader( {
  postCssConfig: {
    path: __dirname,
    ctx: {
      transformCssProperties: isFallbackBuild
    },
  }
} )
```
Then the `postcss.config.js` file can use the context: the `ctx` param is exposed as `options`:
```js
module.exports = ( { options } ) => {
  const { transformCssProperties } = options;
  // return config object
}
```

I think this all is a good tool for this PR. The messy logic stays only at one place (the webpack config).

Reply:
Yup, that's a lot better than the environment variable! 👍